pantheon-bundle/src/main/java/com/redhat/pantheon/asciidoctor/extension/SlingResourceIncludeProcessor.java

```diff
@@ -79,9 +79,13 @@ public void process(Document document, PreprocessorReader reader, String target,
                         .toChild(FileResource::jcrContent)
                         .toField(FileResource.JcrContent::jcrData)
                         .get();
-                content = new StringBuilder(":pantheon_module_id: ")
+                content = new StringBuilder()
+                        .append(":pantheon_module_id: ")
```

Comment:
I wonder if we should change pantheon_module_id to something more generic? :pantheon_document_id?

Reply:
That's probably a solid idea but we'll have to bring @wesruv in to comment on that because it would have downstream impact.

Reply:
I'm not picky, if it's not specific to modules, I'm for it. Better to get in now than later.

Reply:
Sorry! I don't know what I was thinking when I commented yesterday, but yes this *is* module-specific. So let's keep it as-is for now.

(I saw "for now" because when we support assembly-of-assembly, that may change... or it may not... but we'll see when we get there.)